File: task-03/Python/code.py
def checkPrime(n):
    if n < 2:
        return False

    for i in range(2, n):
        if n % i == 0:
            return False

    return True

N = int(input("Enter a number: "))

for i in range(2, N):
    if checkPrime(i):
        print(i)

File: task-03/Haskell/code.hs
checkPrime :: Integer -> Bool
checkPrime n
    | n < 2     = False
    | otherwise = not $ any (\i -> n `mod` i == 0) [2..n-1]

main :: IO ()
main = do
    putStrLn "Enter a number: "
    input <- getLine
    let n = read input :: Integer
    mapM_ (\i -> if checkPrime i then print i else return ()) [2..n-1]
